src/Documentation/Advanced-Concepts/Powershell-Client.md

```diff
@@ -146,7 +150,7 @@ PS> Stop-GrainClient
 
 ```
 
-Thats it for now. We plan to update this page as we introduce more Cmdlets like use Observers, Streams and other Orleans core features more natively on Powershell.
+That is it for now. We plan to update this page as we introduce more Cmdlets like use Observers, Streams and other Orleans core features more natively on Powershell.
```

Comment:
Could we also change the last sentence to: We plan to update this page as we introduce Cmdlets for other Orleans features such as Observers and Streams. ?